package.json

```diff
@@ -43,6 +44,7 @@
     "leaflet-draw": "1.0.4",
     "leaflet.markercluster": "git+https://github.com/liqd/Leaflet.markercluster#5ed89b26922c51083fc9632a2c01425b9261a0f5",
     "lint-staged": "13.0.3",
+    "mapbox-gl-leaflet": "^0.0.15",
```

Comment:
We pin the deps, so the ^ should be removed. If we merge it like that renovate will open a PR, removing it, so that's also an option.